src/index.scss

```diff
@@ -52,6 +52,7 @@ h1 {
 
 h2 {
   line-height: var(--magentaa11y-line-height-43);
+  margin: 2rem 0 1rem;
```

Comment:
did you intend to change ALL h2s across the site? this includes the ones in cards -- I would double check that the margin looks ok to you there!